tests/test_transforms/test_feature_selection/test_filter_transform.py

```diff
@@ -97,3 +89,110 @@ def test_exclude_filter_wrong_column(ts_with_features):
     with pytest.raises(ValueError, match="Features {.*} are not present in the dataset"):
         ts_with_features.fit_transform([transform])
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns, saved_columns",
+    [
+        ([], []),
+        (["target"], ["target"]),
+        (["exog_1", "exog_2"], ["exog_1", "exog_2"]),
+        (["target", "exog_1", "exog_2"], ["target", "exog_1", "exog_2"]),
+    ],
+)
+def test_transform_exclude_save_columns(ts_with_features, columns, saved_columns, return_features):
+    original_df = ts_with_features.to_pandas()
+    transform = FilterFeaturesTransform(exclude=columns, return_features=return_features)
+    ts_with_features.fit_transform([transform])
+    df_transformed = transform._df_removed
+    if return_features:
+        got_columns = set(df_transformed.columns.get_level_values("feature"))
+        assert got_columns == set(saved_columns)
+        for column in got_columns:
+            assert np.all(
+                df_transformed.loc[:, pd.IndexSlice[:, column]] == original_df.loc[:, pd.IndexSlice[:, column]]
+            )
+    else:
+        assert df_transformed is None
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns, saved_columns",
+    [
+        ([], ["target", "exog_1", "exog_2"]),
+        (["target"], ["exog_1", "exog_2"]),
+        (["exog_1", "exog_2"], ["target"]),
+        (["target", "exog_1", "exog_2"], []),
+    ],
+)
+def test_transform_include_save_columns(ts_with_features, columns, saved_columns, return_features):
+    original_df = ts_with_features.to_pandas()
+    transform = FilterFeaturesTransform(include=columns, return_features=return_features)
+    ts_with_features.fit_transform([transform])
+    df_transformed = transform._df_removed
+    if return_features:
+        got_columns = set(df_transformed.columns.get_level_values("feature"))
+        assert got_columns == set(saved_columns)
+        for column in got_columns:
+            assert np.all(
+                df_transformed.loc[:, pd.IndexSlice[:, column]] == original_df.loc[:, pd.IndexSlice[:, column]]
+            )
+    else:
+        assert df_transformed is None
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns",
+    [
+        ([]),
+        (["target"]),
+        (["exog_1", "exog_2"]),
+        (["target", "exog_1", "exog_2"]),
+    ],
+)
+def test_inverse_transform_back_excluded_columns(ts_with_features, columns, return_features):
+    original_df = ts_with_features.to_pandas().copy()
+    transform = FilterFeaturesTransform(exclude=columns, return_features=return_features)
+    ts_with_features.fit_transform([transform])
+    transformed_df = ts_with_features.to_pandas().copy()
+    ts_with_features.inverse_transform()
+    columns_inversed = set(ts_with_features.columns)
+    if return_features:
```

Comment:
You do not actually need ifs here, you just need to parametrize the expected column names in `column_inverse`, and compere the columns in `ts_with_features` with columns of `original_df`
Create parameter `expected_columns`